### PR TITLE
Create find in library dialog

### PR DESCRIPTION
**Pull request by NohYujeong**

### CODE REVIEW

File: app/components/findInLibraryDialog/requestForm.tsx [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+          <span>and let us know about your associated library.</span>
+          <br />
+          <span>We will notify you via email once we’ve integrated with your library.</span>
+        </div>
+      </div>
+      <div className={s.contentDivider} />
+      <div className={s.formWrapper}>
+        <Formik
+          initialValues={{
+            email: '' || currentUser.email,
```
Comment (TylorShin) [inline app/components/findInLibraryDialog/requestForm.tsx]: it always select currentUser.email
if you want to use '' as a fallback, `currentUser.email || ''` is right

File: app/components/findInLibraryDialog/requestForm.tsx [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+            affiliation:
+              '' ||
```
Comment (TylorShin) [inline app/components/findInLibraryDialog/requestForm.tsx]: same here.

File: app/components/findInLibraryDialog/requestForm.tsx [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+          validate={validateForm}
+          onSubmit={handleSubmit}
+          enableReinitialize
+          render={({ errors }) => {
+            return (
+              <Form>
+                <div className={s.fieldWrapper}>
+                  <Field
+                    className={classNames({
+                      [s.emailInput]: true,
```
Comment (TylorShin) [inline app/components/findInLibraryDialog/requestForm.tsx]: don't need to use classNames

File: app/containers/paperShow/reducer.tsx [hunk 4]
```diff
@@ -62,17 +61,8 @@ export function reducer(state: PaperShowState = PAPER_SHOW_INITIAL_STATE, action
...
-      if (requestedAt) {
-        return { ...state, ...{ lastRequestedAt: format(requestedAt, 'MMMM D, YY') } };
-      }
-
-      return { ...state, ...{ lastRequestedAt: requestedAt } };
-    }
-
     case ACTION_TYPES.AUTH_SUCCEEDED_TO_SIGN_OUT: {
-      return { ...state, ...{ lastRequestedAt: null } };
+      return { ...state };
```
Comment (TylorShin) [inline app/containers/paperShow/reducer.tsx]: you can delete the case itself.

File: app/components/findInLibraryDialog/requestForm.tsx [hunk 5]
```diff
@@ -49,15 +49,17 @@ const RequestForm: React.FC<{
...
       </div>
       <div className={s.contentsArea}>
         <div>
-          <span>Please fill out the form below.</span>
+          <span>Fill out the form below</span>
           <br />
-          <span>We will notify you by email after integrating based on the information you submit.</span>
+          <span>and let us know about your associated library.</span>
+          <br />
+          <span>We will notify you via email once we’ve integrated with your library.</span>
```
[the remark this answers is not in the document]
Comment (NohYujeong) [reply app/components/findInLibraryDialog/requestForm.tsx]: I thought there was a big difference, but there is no difference(from june).

So, to unite in one word, I changed `Integrated` to `linked`.

File: app/components/findAtLibraryDialog/requestForm.tsx [hunk 6]
```diff
@@ -0,0 +1,125 @@
...
+        </div>
+      </div>
+      <div className={s.contentDivider} />
+      <div className={s.formWrapper}>
+        <Formik
+          initialValues={{
+            email: '' || currentUser.email,
+            affiliation:
+              '' ||
+              ({ id: currentUser.affiliationId, name: currentUser.affiliationName, nameAbbrev: null } as Affiliation),
```
[the remark this answers is not in the document]
Comment (NohYujeong) [reply app/components/findAtLibraryDialog/requestForm.tsx]: That's my mistake.

My edit will be reflected soon.

**Pull request merged**